audit/projects/k8s-release/buckets/k8s-release-pull/iam.json

```diff
@@ -3,6 +3,7 @@
     {
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
+        "group:k8s-infra-prow-oncall@kubernetes.io",
```

Comment:
Expected, all the IAM changes to the `k8s-release-dev` buckets are part of https://github.com/kubernetes/k8s.io/pull/2333